bip-txrelayv2.mediawiki

```diff
@@ -0,0 +1,49 @@
+<pre>
+  BIP: XXX
+  Layer: Peer Services
+  Title: Transaction Relay V2
```

Comment:
Could you perhaps reconsider the title? This sounds somewhat similar to the v2 P2P protocol that was recently deployed. Maybe something along the lines of "Behavior regarding unsolicited Transactions", if I’m getting the gist of what you are thinking about?

Reply:
This is intentional to be similar to BIP24 and the upgrade of the V2 P2P protocol. Before the BIP design and implementation, the peer-to-peer protocol was not versioned, neither really encrypted. Similarly, we’re introducing a versioning of the transaction-relay protocol, which a node can use to selectively support policies and mechanisms.